interface/storage/file_sharing/samba_ops.rst

```diff
@@ -1,138 +1,339 @@
+*media_user*, we will enter its login and password:
+
+.. image:: /images/interface/storage/file_sharing/samba_ops/samba_access_win_login_prompt.png
+   :align: center
+
+
+.. note::
+
+  The domain name is defaulted to your machine name. Enter :code:`DOMAINNAME\username`
+  in the username field to use an alternative domain name (such as an AD
+  domain, for instance).
+
+Check *Remember my credentials*, and click OK. Windows Explorer will then
+attempt to connect to the Rockstor appliance, and if successful, you will be
+able to see the content of the *Photos* share available on Rockstor.
 
-From a Mac:
-------------
+.. image:: /images/interface/storage/file_sharing/samba_ops/samba_access_win_success.png
+   :width: 100%
+   :align: center
+
+
+Adding a network location
+"""""""""""""""""""""""""
+
+In the Windows Explorer menu, click *Add a network location*. This will open a
+dedicated wizard. When asked for the address of the network location, enter the
+address of the Rockstor share using the following format:
+
+    \\\\<rockstor-appliance-ip>\\<share-name>
+
+In our *Photos* share example, this corresponds to:
+
+.. image:: /images/interface/storage/file_sharing/samba_ops/samba_access_win_network_location_wizard_path.png
+   :align: center
+
+
+Click *Next*. When asked for credentials, use the corresponding username and
+password. As we set up the *Photos* share to be accessed by the user named
+*media_user*, we will enter its login and password:
+
+.. image:: /images/interface/storage/file_sharing/samba_ops/samba_access_win_login_prompt.png
+   :align: center
+
+
+.. note::
+
+  The domain name is defaulted to your machine name. Enter :code:`DOMAINNAME\username`
+  in the username field to use an alternative domain name (such as an AD
+  domain, for instance).
+
+Check *Remember my credentials*, and click OK.
+
+You will then be asked to enter the name of your choice for this share. This
+will be the *display* name of this share in Windows and will not affect the
+name of the share in Rockstor. You can thus enter any name or description you
+would like. We thus recommend using a short and explicit name. In our *Photos*
+example, this could thus be:
+
+.. image:: /images/interface/storage/file_sharing/samba_ops/samba_access_win_network_location_wizard_name.png
+   :align: center
+
+
+Click *Next* and complete the wizard; you should now be able to copy to and
+access files on this share.
+
+
+Map a network drive
+"""""""""""""""""""
 
-1. On your Mac, open the Finder, and select *Go* -> *Connect to Server*
+In the Windows Explorer menu, click *Map network drive*. This will open a
+dedicated wizard. When asked for the address of the network location, enter the
+address of the Rockstor share using the following format:
 
-2. In the Server address field, type in the address of the shared folder as
-   shown below (with the ip address of your Rockstor appliance)
+    \\\\<rockstor-appliance-ip>\\<share-name>
```

Comment:
Same again here re the rendering. I get:

>     \<rockstor-appliance-ip>\<share-name>

Reply:
Curious... It looked correct when testing using `python -m http.server ...`:
![image](https://user-images.githubusercontent.com/30297881/147284251-1f70b500-19bf-4dbc-bc87-2df15cfa2a91.png)

I suppose you checked under apache2?

Reply:
Yes, via the buildbot "PUBLISH-Docs" Builder with local target and this pr number entered.